tsdb/chunkenc/histogram_meta.go

```diff
@@ -628,7 +544,7 @@ func adjustForInserts(spans []histogram.Span, inserts []Insert) (mergedSpans []h
 		}
 	}
 	for i < len(inserts) {
-		addBucket(inserts[i].bucketIdx)
+		addBucket(insertIdx)
```

Comment:
Note to reviewers: this only worked because we didn't have multiple inserts at the end.